ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -885,6 +895,17 @@ MINFC:
   - bypass_valve_percentage_command
   - min_flowrate_setpoint
   - flowrate_sensor
+  
+MAXFC:
+  id: "16039155144679489536"
+  description: "Maximum flow control for entire loop."
+  is_abstract: true
+  implements:
+  - CONTROL
+  uses:
+  - bypass_valve_percentage_command
+  - max_flowrate_setpoint
```

Comment:
unless you are aggregating a bunch of setpoints, this is not a `max_flowrate_setpoint` -- its a high-limit flowrate setpoint. please adjust. i am also confused why you would be controlling to high limit; bypass valves are usually for low-limit flows (that is, to keep the flow in the primary loop above a certain minimum value). what is the control strategy here?